python_scripts/ensemble_gradient_boosting.py

```diff
@@ -193,10 +194,10 @@ def generate_data(n_samples=50):
 # and compare it with the true value.
 
 # %%
-print(f"True value to predict for f(x={data_max:.3f}) = {target_true:.3f}")
+print(f"True value to predict for f(x={data_max.Feature[0]:.3f}) = {target_true:.3f}")
```

Comment:
To be consistent with the other part of the code

```suggestion
print(f"True value to predict for f(x={data_max.["Feature"][0]:.3f}) = {target_true:.3f}")
```

Can you make the same change below